src/index.html

```diff
@@ -11,7 +11,184 @@
+          <input
+            type="date"
+            name="date"
+            class="field"
+          >
+        </label>
+
+        <label>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="Iacceptthetermoftheagreement"
+            class="field"
+          >
+        </label>
+
+      </fieldset>
+      <fieldset enabled class="forma">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            class="field"
+            required
+            placeholder="email@example.com"
+          >
+        </label>
+        <label>
+          Password:
+          <input
+            type="password"
+            name="password"
+            class="field"
+            autocomplete="off"
+          >
+        </label>
+
+      </fieldset>
+      <fieldset enabled class="forma">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          <label>
+            Do you love cats?
+            <input
+              type="radio"
+              name="answer"
+              class="field"
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label >
+            <input
+              type="radio"
+              name="answer"
+              class="field"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+
+
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            class="field"
+          >
+        </label>
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+            class="field"
+          >
+        </label>
+        <label class="form-field">
+          What are your favorite brand of cars?
+
+          <select
+            name="brand"
+            multiple
+            class="field"
+            >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label>
+        How do you rate our work?
+          <input
+            type="range"
+            name="rate"
+            class="field"
+            value="0"
+            min="0"
+            max="30"
+            step="1"
+          >
+        </label>
+
+      </fieldset>
+      <fieldset class="forma">
+        <legend>Additional info:</legend>
+
+        <label class="form-field">
+          Comments:
+          <textarea
+            name="comments"
+            class="field">
+          </textarea>
+        </label>
+
+        <label>
+            Would you recommend us?
+          <select
+             name="answer"
+             class="field"
+          >
```

Comment:
you can write the tag in one line if the attributes number is less than 3
![image](https://user-images.githubusercontent.com/22824431/227933219-b48dd3ef-a82f-4307-8d9b-b7dec6d2102b.png)

```suggestion
          <select name="answer" class="field">
```